src/DTO/DefaultProxyFeature.php

```diff
@@ -37,12 +37,12 @@ public function __construct(array $response)
         // tries to new this up manually and isn't interesting to tests
 
         // @codeCoverageIgnoreStart
-        assert(isset($response['name'], $response['enabled'], $response['variant'], $response['impression_data']));
+        assert(isset($response['name'], $response['enabled'], $response['variant'], $response['impressionData']));
```

Comment:
This will fail in dev, if using with older version of Unleash (possibly in production as well, if it's misconfigured).

Reply:
Yes, changing the assert